### DISCUSSION TITLE
Which sources should be added next?

### DISCUSSION THREAD

**Discussion by ricoberger** [feeddeck/feeddeck#80] [Polls]
Please let me know which sources are you missing in FeedDeck. You can vote for the source you want to see next and you can also provide additional sources within the comments.

**Note:** Only because a source is listed here does not mean that it will be added. Whether a source is added depends on whether it provides an RSS feed or API to scrape the content of the source.

**Comment by 23hp** [comment]
Anyone watch Tagesspiegel?

**Reply by ricoberger** [reply]
Hi @23hp, for Tagesspiegel the following RSS feeds should be working:

- https://www.tagesspiegel.de/contentexport/feed/home
- https://www.tagesspiegel.de/contentexport/feed/politik

**Reply by 23hp** [reply]
Thank you Rico, I'll try!
